### ISSUE TITLE
Enhancement: Gemeinschaftskonto

### ISSUE THREAD

**Issue by danielkastner** [buchen/portfolio#1172]
Ich fände es gut wenn man bei Konten bzw. Depots einen Faktor angeben könnte der festlegt, mit welchem Anteil der Kontostand bzw. Depotwert in die Berechnung des Vermögens, Performance etc. eingeht. 

Hintergrund ist, dass meine Partnerin und ich gemeinschaftliche Konten - und in Zukunft ggf. gemeinsame Depots haben - mich aber nur "mein Anteil" daran interessiert. 

Wenn auf unserem gemeinsamen Konto also EUR 5327.08 sind, würde ich gerne in die Aufstellung bzw. Berechnung der Performance nur EUR 2663.54 übernehmen.

**Comment by Ragas13** [comment]
@danielkastner
Du meinst das du alle Beträge optional durch 2 gerechnet bzw angezeigt werden sollen?

Abgesehen davon das es eigentlich am Sinn und Zweck vom **Gemeinschafts**konten / -depot vorbei geht, dass Ziel von PP ist eigentlich das Aggregieren von Konten und Depots und nicht das aufteilen von Vermögenswerten. Darum kümmern sich später i.d.R. Anwälte, aber (das ist meine persönliche Meinung) bis dahin sollte eigentlich "schau mal Schatz, dass gehört **uns**" im Vordergrund stehen.

**Comment by danielkastner** [comment]
@Ragas13 Ja, genau sowas meine ich. Wobei ich nicht alle Beträge meine sondern nur den Wert eines Depots bzw. das Kontoguthaben. Einzelne Buchungen etc. sollen unverändert angezeigt werden.

**Comment by buchen** [comment]
Könntest Du nicht eine Klassifizierung anlegen, z.B. "Meine" und "Ihre" und das Konto zu je 50% den beiden Kategorien zuordnen. Die Aktien ordnest Du dann der Kategorie "Meine" hinzu. Diese Kategorie verwendest Du dann in den Auswertungen statt der Datenreihe "Gesamtportfolio".

Das sollte solange funktionieren bis Ihr die gleichen Wertpapiere kauft. Wenn Ihr Wertpapiere genau 50:50 kauft, könnte man das auch abbilden.

Ansonsten sollte man zwei verschiedene Depots anlegen (ggf. auf das selbe Verrechnungskonto). Allerdings kann man den Diagrammen entweder eine Datenreihe auf Depot/Konto Kombination mitgeben oder auf Klassifikation - aber nicht beides.
